diff --git a/cogdb/scrape.py b/cogdb/scrape.py
index 9d30e5d..c55dfba 100644
--- a/cogdb/scrape.py
+++ b/cogdb/scrape.py
@@ -159,17 +159,19 @@ def powerplay_leader(driver, leader_index, *, updated_at=0, held_merits=False):
     time.sleep(LONG_GAP)
 
     # Select leader from drop down by index, sometimes delayed
-    retries = 5
-    while retries:
-        try:
-            selects = driver.find_elements(By.CLASS_NAME, "blazored-typeahead__result ")
-            click_with_retry(selects[leader_index])
-        except IndexError:
-            if retries == 0:
-                raise
-            time.sleep(LONG_GAP)
-        finally:
-            retries = retries - 1
+    #  retries = 5
+    #  while retries:
+        #  try:
+            #  selects = driver.find_elements(By.CLASS_NAME, "blazored-typeahead__result ")
+            #  click_with_retry(selects[leader_index])
+        #  except IndexError:
+            #  if retries == 0:
+                #  raise
+            #  time.sleep(LONG_GAP)
+        #  finally:
+            #  retries = retries - 1
+    selects = driver.find_elements(By.CLASS_NAME, "blazored-typeahead__result ")
+    click_with_retry(selects[leader_index])
     now = datetime.datetime.utcnow()  # Time is relative when you click leader in list
     time.sleep(LONG_GAP)
 
